wallets/blink/common.js

```diff
@@ -37,15 +39,14 @@ export async function request (authToken, query, variables = {}) {
     body: JSON.stringify({ query, variables })
   }
   const res = await fetch(galoyBlinkUrl, options)
-  if (res.status >= 400 && res.status <= 599) {
-    // consume res
-    res.text().catch(() => {})
-    if (res.status === 401) {
-      throw new Error('unauthorized')
-    } else {
-      throw new Error('API responded with HTTP ' + res.status)
-    }
-  }
+
+  // consume response body to avoid memory leaks
+  // see https://github.com/nodejs/node/issues/51162
+  res.text().catch(() => {})
```

Comment:
@riccardobl I remember you mentioned memory leaks if the body is not consumed as mentioned in https://github.com/nodejs/node/issues/51162. But the ticket is closed so I am not sure if that's a problem? Afaict, the response body will be garbage-collected.

If this is still a problem, shouldn't we make sure the body is consumed for every fetch on the server?

Reply:
Oh, I think I added a bug since I think I consume the body here so I no longer have it for `res.json()`.

Reply:
> This is still a problem and afaik we do consume the body on all fetches on the server.

Mhh, but not in the error handling I added in #1516. We can throw before we consume it.